### PR TITLE
Fix/added loader to all pages

### PR DESCRIPTION
**Pull request by rayy40**
### Fixes

Use common loader across all the pages of the application
Reference - #1494

### Solution

For the pages that lacked a loading state, the loading state was created, and a 'Loader' was added to display the loading.
Additionally, the pages that were already in a loading state were changed to use the proper 'Loader' style.

```
import Loader from "common/Loader/index"

<div className="flex h-full w-full items-center justify-center">
      <Loader />
</div>
```

### CODE REVIEW

**Comment by rayy40** [comment]
Hey @prasanthchaduvula I have raised the PR for the issue #1494 
Can you please review it and let me know if I should make any changes?

**Review by akhilgkrishnan — COMMENTED** [review]
Thanks @rayy40 for working on this 👍 .
If possible could you please record a demo video for the output.

File: app/javascript/src/components/payments/Success/index.tsx [hunk 1]
```diff
@@ -33,7 +33,11 @@ const Success = () => {
   }, []);
 
   if (loading) {
-    return <Loader />;
+    return (
+      <div className="flex h-80v w-full flex-col justify-center">
+        <Loader />
+      </div>
```
Comment (akhilgkrishnan) [inline app/javascript/src/components/payments/Success/index.tsx:39]: Seems like we are using the same div classes for all the loaders. Can we move this inside to the loader component?

Comment (prasanthchaduvula) [reply app/javascript/src/components/payments/Success/index.tsx:39]: @rayy40 Please move the common classNames to Loader components and also add a customization option by adding className props to the Loader component which will add the extra classes to the Loader parent div and can be used to override default classnames.

Comment (rayy40) [reply app/javascript/src/components/payments/Success/index.tsx:39]: Okay sure, I will get on with it.

**Comment by rayy40** [comment]
@akhilgkrishnan , I have updated the changes.
Can you please take a look at it?

**Comment by akhilgkrishnan** [comment]
@apoorv1316 Could you please test this PR?

**Comment by supriya3105** [comment]
@apoorv1316 Have you tested this PR ?

**Comment by apoorv1316** [comment]
@prasanthchaduvula We have an empty state for reports and a few other pages that appear while the page is loaded and in this PR we are using the 'Miru Loader' on almost all pages. It's not consistent. Let's try to make it more consistent across the app.

**Comment by akhilgkrishnan** [comment]
> @prasanthchaduvula We have an empty state for reports and a few other pages that appear while the page is loaded and in this PR we are using the 'Miru Loader' on almost all pages. It's not consistent. Let's try to make it more consistent across the app.

If possible can you list all the pages?

**Comment by rayy40** [comment]
@apoorv1316 @akhilgkrishnan , Hey guys any updates?

**Comment by akhilgkrishnan** [comment]
> @apoorv1316 @akhilgkrishnan , Hey guys any updates?

@apoorv1316 We can create a separate ticket adding the loader for missing pages. Could you please check the current implementation, if everything looks good we can merge.

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**